internal/scanner/scanner.go

```diff
@@ -2,6 +2,8 @@ package scanner
 
 import (
 	"context"
+	"fmt"
+	"github.com/gorilla/websocket"
```

Comment:
Please fix the order of the packages. All non-standard packages must be below

Reply:
Fxd with goimports + gofmt utils